src/components/RangeSeries.tsx

```diff
@@ -26,27 +27,47 @@ export function RangeSeries<T extends RangeSeriesPoint>(
 ) {
   const id = useId(props.id, 'series');
   const [, legendDispatch] = useLegend();
-  const { lineStyle = {}, hidden, xAxis, yAxis, data, label } = props;
-
+  const {
+    lineStyle = {},
+    hidden,
+    xAxis = 'x',
+    yAxis = 'y',
+    data,
+    label,
+    xShift: oldXShift = '0',
+    yShift: oldYShift = '0',
+  } = props;
+
+  const { xShift, yShift } = useShift({
+    horizontalAxisId: xAxis,
+    verticalAxisId: yAxis,
+    xShift: oldXShift,
+    yShift: oldYShift,
+  });
   // Update plot context with data description
   const dispatch = usePlotDispatchContext();
+  const { xShift: xShiftInverted, yShift: yShiftInverted } = useInvertShift({
+    xShift,
+    yShift,
```

Comment:
shouldn't the axis be passed here as well?

I expect there to be a bug if the default axis are not used

Reply:
Yes true thank you